ansible/roles/es-mapping/files/mappings/userv3-mapping.json

```diff
@@ -889,6 +889,937 @@
+              "fields": {
+                "raw": {
+                  "type": "text",
+                  "analyzer": "keylower",
+                  "fielddata": true
+                }
+              },
+              "copy_to": [
+                "all_fields"
+              ],
+              "analyzer": "cs_index_analyzer",
+              "search_analyzer": "cs_search_analyzer",
+              "fielddata": true
+            },
+            "middlename": {
+              "type": "text",
+              "fields": {
+                "raw": {
+                  "type": "text",
+                  "analyzer": "keylower",
+                  "fielddata": true
+                }
+              },
+              "copy_to": [
+                "all_fields"
+              ],
+              "analyzer": "cs_index_analyzer",
+              "search_analyzer": "cs_search_analyzer",
+              "fielddata": true
+            },
+            "surname": {
+              "type": "text",
+              "fields": {
+                "raw": {
+                  "type": "text",
+                  "analyzer": "keylower",
+                  "fielddata": true
+                }
+              },
+              "copy_to": [
+                "all_fields"
+              ],
+              "analyzer": "cs_index_analyzer",
+              "search_analyzer": "cs_search_analyzer",
+              "fielddata": true
+            },
+            "dob": {
+              "type": "text",
+              "fields": {
+                "raw": {
+                  "type": "text",
+                  "analyzer": "keylower",
+                  "fielddata": true
+                }
+              },
+              "copy_to": [
+                "all_fields"
+              ],
+              "analyzer": "cs_index_analyzer",
+              "search_analyzer": "cs_search_analyzer",
+              "fielddata": true
+            },
+            "maritalStatus": {
+              "type": "text",
+              "fields": {
+                "raw": {
+                  "type": "text",
+                  "analyzer": "keylower",
+                  "fielddata": true
+                }
+              },
+              "copy_to": [
+                "all_fields"
+              ],
+              "analyzer": "cs_index_analyzer",
+              "search_analyzer": "cs_search_analyzer",
+              "fielddata": true
+            },
+            "category": {
+              "type": "text",
+              "fields": {
+                "raw": {
+                  "type": "text",
+                  "analyzer": "keylower",
+                  "fielddata": true
+                }
+              },
+              "copy_to": [
+                "all_fields"
+              ],
+              "analyzer": "cs_index_analyzer",
+              "search_analyzer": "cs_search_analyzer",
+              "fielddata": true
+            },
+            "nationality": {
+              "type": "text",
+              "fields": {
```

Comment:
Please take a look again for all these fields and change the data type for fields if its not required to analyzed